appendix-basic65.tex

```diff
@@ -9218,8 +9335,7 @@ \subsection{SGN}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$B4
 \item [Format:] {\bf SGN(}numeric expression{\bf)}
-\item [Usage:] Extracts the sign from
-               the argument and returns it as a number:
+\item [Usage:] The sign of a numeric expression, as a number.
 
                 \begin{itemize}
                     \item {\bf -1} negative argument.
```

Comment:
> \item {\bf -0} zero.

Trying it on a recent rom, "PRINT SGN(0)" returns 0 (not -0). I haven't checked with older roms as yet.

Reply:
Looks like an old typo, there's no such thing as -0 in BASIC. Fixed.